udata_front/theme/gouvfr/assets/js/pages/ReusePublishingForm/Step2AddDatasets.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <Container>
+    <Stepper :steps="steps" :currentStep="1"/>
+    <Well
+      color="blue-cumulus"
+      weight="regular"
+      class="fr-mb-2w"
+    >
+      <div class="fr-grid-row">
+        <div class="fr-col-auto fr-mr-3v">
+          <span class="fr-icon-info-line" aria-hidden="true"></span>
+        </div>
+        <div class="fr-col">
+          <p class="fr-m-0 fr-text--xs">{{ t('It is important to associate all the datasets used, as this allows to understand the cross-references and improve the visibility of your reuse.') }}</p>
+        </div>
+      </div>
+    </Well>
+    <fieldset class="fr-fieldset min-width-0" aria-labelledby="description-legend">
+      <legend class="fr-fieldset__legend" id="description-legend" v-if="form.datasets.length > 0">
+        <h2 class="subtitle subtitle--uppercase fr-mb-0">
+          {{ t("Associated datasets") }}
+        </h2>
+      </legend>
+      <div ref="el">
+        <div class="fr-col fr-grid-row fr-grid-row--center w-100" v-for="(dataset, index) in form.datasets" :key="dataset.id">
+          <div class="fr-col-auto fr-my-auto fr-ml-auto justify-center flex fr-mr-3v">
+            <img :src="draggableIcon" />
+          </div>
+          <div class="fr-col relative">
```

Comment:
```suggestion
          <div class="fr-col relative">
```
add a function to compute a z-index style like the search page